python/ray/data/dataset.py

```diff
@@ -793,7 +793,7 @@ def randomize_block_order(
         *,
         seed: Optional[int] = None,
     ) -> "Dataset[T]":
-        """Randomly shuffle the blocks of this dataset.
+        """Randomly shuffle the blocks of this dataset (EXPERIMENTAL).
```

Comment:
Out of curiosity, do we have a clear distinction between experimental APIs and Alpha APIs?

Reply:
Hmm probably alpha if we have more confidence in it's functionality and are just concerned about API stability. This one I think probably right now the quality is not there.